content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -737,7 +737,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
   [readiness probes](/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes).
 - `ExpandCSIVolumes`: Enable the expanding of CSI volumes.
 - `ExpandedDNSConfig`: Enable kubelet and kube-apiserver to allow more DNS
-  search paths and longer list of DNS search paths. See
+  search paths and longer list of DNS search paths. This feature requires container
+  runtime support(Containerd: v1.5.6 or higher, CRI-O: v1.22 or higher). See
```

Comment:
This will be enabled by default and will break people , or is it safe to consider that every cluster will have these versions?

Reply:
https://github.com/kubernetes/enhancements/pull/2884#pullrequestreview-747869929

I think it might be safe to enable this by default after all unsupported container runtimes are EOL.